ci/cron/src/Main.hs

```diff
@@ -268,11 +269,19 @@ docs = do
             update_s3 temp_dir gh_versions
         reset_cloudfront
 
+bash :: Maybe FilePath -> [String] -> IO String
+bash bash_lib cmd = do
+    shell $ unlines $ concat [
+        ["bash -c '",
+         "set -euo pipefail",
+         "eval \"$(dev-env/bin/dade assist)\"",
```

Comment:
Do we actually need this? I thought this just sets a bunch of environment variables so if we assume that this script runs in dev-env (which seems like a very reasonable assumption to me), we should be able to skip this.

Reply:
True, we should be able to skip this.